File: Builder Java/Casa.java
public class Casa {
    String MiCasa=null;

    Casa(){
        MiCasa="";
    }

    public Casa Techo(){
        this.MiCasa+="Techo ";
        return this;
    }
    public Casa Pared(){
        this.MiCasa+="Pared ";
        return this;
    }
    public Casa Puerta(){
        this.MiCasa+="Puerta ";
        return this;
    }
    public void printHouse(){
        System.out.println(MiCasa);
    }
}

File: Builder Python/Casa.py
class Casa:
    CasaBuild=[]

    def __init__(self):
        self.CasaBuild=list()

    def setPuerta(self):
        self.CasaBuild.append('Puerta')
        return self

    def setTecho(self):
        self.CasaBuild.append('Techo')
        return self

    def setPared(self):
        self.CasaBuild.append('Pared')
        return self

    def PrintCasa(self):
        return self.CasaBuild